minotaur.pro

```diff
@@ -9,7 +9,7 @@ QT       += core gui
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 
 # Use C++11 compiler
-QMAKE_CXXFLAGS += -std=c++11
+#QMAKE_CXXFLAGS += -std=c++11

```

Comment:
i forgot if this was addressed before, but what was the reason again for removing this?